a10_octavia/controller/worker/tasks/health_monitor_tasks.py

```diff
@@ -122,11 +124,12 @@ def execute(self, listeners, health_mon, vthunder, update_dict):
             expect_code = health_mon.expected_codes
         args = utils.meta(health_mon, 'hm', {})
         try:
+            post_data = CONF.a10_health_manager.post_data
```

Comment:
Is it alright to take this `post_data` configured from  `[a10_health_manager]` section? Because according to me, `health monitor` created via openstack cli is part of `a10-controller-worker.service` service whereas, things configured under `[a10_health_manager]` section is for separate service called `a10-health_manager.service`

I would suggest we should read it from another section called `[health_monitor]` similar to  `[listener]`, `[slb]` etc. And from there we would read `post_data`

Reply:
```
[health_monitor]
post_data = "abc=1"
```

Reply:
changed as the comment